bitnami/kube-state-metrics/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 1.9.7
 description: kube-state-metrics is a simple service that listens to the Kubernetes API server and generates metrics about the state of the objects.
 name: kube-state-metrics
-version: 0.3.2
+version: 0.3.3
```

Comment:
Adding a new parameter requires bumping the minor version

Reply:
Yes you are right.